cub/cub/thread/thread_reduce.cuh

```diff
@@ -43,15 +48,62 @@
 #  pragma system_header
 #endif // no system header
 
-#include <cub/detail/type_traits.cuh>
-#include <cub/thread/thread_operators.cuh>
+#include <cub/detail/type_traits.cuh>        // are_same_v
+#include <cub/thread/thread_operators.cuh>  // DpxMin
+#include <cub/util_namespace.cuh>
+#include <cub/util_type.cuh>
 
 CUB_NAMESPACE_BEGIN
 
 /// Internal namespace (to prevent ADL mishaps between static functions when mixing different CUB installations)
 namespace internal
 {
 
+//----------------------------------------------------------------------------------------------------------------------
+
+/// Enable DPX reduction if the following conditions are met:
+/// - Hopper+ architectures. DPX instructions are emulated before Hopper.
+/// - The number of elements must be large enough for performance reasons (see below)
+/// - All types must be the same
+/// - Only works with integral types of 1 bytes or 2 bytes
```

Comment:
The comment of 1 byte does not seem to match what is in the code below. Only int16_t and uint16_t seem to be enabled.